articles/configure-external-dns.mdx

```diff
@@ -0,0 +1,122 @@
+
+  If you need help updating your DNS, please contact our friendly [support team](/support/).
+</comp.Notice>
+
+To switch to External DNS:
+
+1. Navigate to the *Domain browser* by clicking the globe icon in your *App Navigation*.
+2. Select the domain for which you want to configure DNS.
+3. Under the *Domain Settings* tab, open the *DNS* section.
+4. Select the *External DNS* option in the *DNS Provider* dropdown.
+5. Click the *Update DNS Provider* button.
+
+You can now manage your DNS records through an external provider.
+
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/hosting/configure-external-dns/2024-12/CloudCannon-Documentation-Domain-Settings-External-DNS.png" alt="A screenshot of the Domain Settings page shows the DNS Provider dropdown is set to External DNS." title="The Domain Settings page" type="screenshot"/>
+
+### Load balancing on external DNS
+
+x
+
+<comp.Notice info_type="important">
+  [CloudCannon DNS](/documentation/articles/configure-cloudcannon-dns/) is strongly recommended for sites on apex domains (no subdomain).\* External DNS\* relies on an A record mapped to a single server. This cannot be dynamically load balanced, reducing redundancy and increasing potential down time risk.
+</comp.Notice>
+
+### CloudCannon DNS records
+
+When you enable external DNS for a Site hosted on CloudCannon, CloudCannon will ask you to add three DNS records to your DNS provider: two A Records and a CNAME Record.
+
+The two A Records specify the IP addresses of the appropriate CloudCannon hosting servers for the your base domain. The CNAME Record will direct queries for any subdomains to CloudCannon's hosting servers.
+
+<comp.Notice info_type="important">
+  If you use a Cloudflare, setting up a CNAME Record for your base domain while using external DNS will result in a 1001 DNS Error.
+</comp.Notice>
+
+## Configure Cloudflare Proxy
+
+CloudCannon hosting uses Cloudflare. If you are using Cloudflare as your DNS provider and have Cloudflare's Orange Cloud feature enabled for your DNS records, you will not be able to proxy directly to the normal CloudCannon CNAME (i.e., site.cloudcannon.com).
+
+To resolve this issue, you can either turn off Orange Cloud, or set up Cloudflare Proxy with CloudCannon Origin and add a CNAME record to your DNS provider.
+
+To set up Cloudflare Proxy with CloudCannon Origin:
+
+1. Navigate to the *Custom Domain* page, under *Site Settings*.
+2. Under the *Setup* tab, select the *Cloudflare Proxy with CloudCannon Origin* option in the *Hosting Setup* dropdown.
+3. Click the *Switch to Cloudflare Proxy with CloudCannon Origin* to confirm.
+
+To add a CNAME record:
+
+1. Log in to your third party DNS provider.
+2. Add a CNAME record pointing your Domain Name (e.g., example.com) at *orange-cloud.cloudcannon.com*.
```

Comment:
`orange-cloud.cloudcannon.com` won't be updated with the new `proxied/dns only` terms update